src/dotnet/commands/dotnet-restore3/Restore3CommandApp.cs

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using Microsoft.DotNet.Cli;
+using Microsoft.DotNet.Cli.CommandLine;
+
+namespace Microsoft.DotNet.Tools.Restore3
+{
+    public class Restore3CommandApp
+    {
+        public static int Execute(string[] args)
+        {
+            CommandLineApplication cmd = new CommandLineApplication
+            {
+                Name = "restore3",
+                FullName = "restore3",
+                Description = "restore for msbuild"
+            };
+
+            cmd.HelpOption("-h|--help");
+
+            var argRoot = cmd.Argument(
+                    "[root]",
+                    "Optional path to a project file or MSBuild arguments.",
+                    multipleValues: true);
+
+            var sourceOption = cmd.Option(
```

Comment:
FallbackSource has been removed, I know it's used some places in the CLI build still. Is there a good reason to keep this instead of just having the user specify all sources with ``-s``?

``--verbosity`` ended up being a converter from the nuget level names to the msbuild level names, but going forward it seemed really unnecessary. Going forward I think this command should use whatever verbosity setting the rest of the commands use, and for now ``/v:`` lets you set it directly on msbuild, which controls the nuget restore output.